typescript/web/src/components/import-button/import-images-modal/modal-dropzone/import-dropped-files/import-datasets.tsx

```diff
@@ -0,0 +1,95 @@
+import { ApolloClient, gql } from "@apollo/client";
+import { ExportFormat } from "@labelflow/graphql-types";
+import { v4 as uuidv4 } from "uuid";
+import mime from "mime-types";
+
+import Bluebird from "bluebird";
+import { uploadFile } from "../../../../../utils/upload-file";
+import { DroppedFile, SetUploadStatuses } from "../../types";
+
+import { CONCURRENCY } from "../../constants";
+
+const importDatasetMutation = gql`
+  mutation importDataset(
+    $where: DatasetWhereUniqueInput!
+    $data: DatasetImportInput!
+  ) {
+    importDataset(where: $where, data: $data) {
+      error
+    }
+  }
+`;
+const importDataset = async ({
+  apolloClient,
+  datasetId,
+  workspaceId,
+  file,
+}: {
+  apolloClient: ApolloClient<any>;
```

Comment:
Can't we give ApolloClient the correct type to infer? (or use the `useMutation` hook)

Reply:
Done in 3438be54a3dff8dd5d3955943810b3d7416796b7